lib/rules/indent_style.ts

```diff
@@ -18,13 +18,7 @@ function resolve(settings: eclint.Settings) {
 }
 
 function check(settings: eclint.Settings, line: linez.Line) {
-	function creatErroe(message: string, columnNumber: number = 1) {
-		message = message.replace(/\b(\d+)\s.+?$/, function(str, count) {
-			if (+count > 1) {
-				str += 's';
-			}
-			return str;
-		});
+	function createError(message: string, columnNumber: number = 1) {
```

Comment:
Sorry, I should have caught this sooner, but you also don't need to declare a type here, as it will be inferred by your default value of `1` to be a `number`. Example:
```ts
function foo(bar = 1) {
  return bar; // number
}
```